shpc/main/modules/templates/singularity.tcl

```diff
@@ -72,23 +73,23 @@ set runCmd "singularity \${SINGULARITY_OPTS} run \${SINGULARITY_COMMAND_OPTS} {%
 set inspectCmd "singularity \${SINGULARITY_OPTS} inspect \${SINGULARITY_COMMAND_OPTS} " 
 
 # set_shell_function takes bashStr and cshStr
-set-alias "{|module_name|}-shell" "${shellCmd} $*"
+set-alias {|module_name|}-shell "${shellCmd}"
 
 
 # exec functions to provide "alias" to module commands
 {% if aliases %}{% for alias in aliases %}
-set-alias "{{ alias.name }}" "${execCmd} {% if alias.options %} {{ alias.options }} {% endif %} ${containerPath} {{ alias.command }} $*"
+set-alias {{ alias.name }} "${execCmd} {% if alias.options %} {{ alias.options }} {% endif %} ${containerPath} {{ alias.command }}"
 {% endfor %}{% endif %}
 
 # A customizable exec function
-set-alias "{|module_name|}-exec" "${execCmd} ${containerPath} \${SINGULARITY_COMMAND_ARGS} $*"
+set-alias {|module_name|}-exec "${execCmd} ${containerPath} \${SINGULARITY_COMMAND_ARGS}"
```

Comment:
Here as well, if the user wants to give args to the alias they can just be provided! (This was also my mistake, sorry about that!)

```suggestion
set-alias {|module_name|}-exec "${execCmd} ${containerPath}"
```